operator/v1/tests/etcds.operator.openshift.io/AAA_ungated.yaml

```diff
@@ -1,16 +1,16 @@
-apiVersion: apiextensions.k8s.io/v1 # Hack because controller-gen complains if we don't have this
-name: "Etcd"
-crdName: etcds.operator.openshift.io
-tests:
-  onCreate:
-    - name: Should be able to create a minimal Etcd
-      initial: |
-        apiVersion: operator.openshift.io/v1
-        kind: Etcd
-        spec: {} # No spec is required for a Etcd
-      expected: |
-        apiVersion: operator.openshift.io/v1
-        kind: Etcd
-        spec:
-          logLevel: Normal
-          operatorLogLevel: Normal
+#apiVersion: apiextensions.k8s.io/v1 # Hack because controller-gen complains if we don't have this
+#name: "Etcd"
+#crdName: etcds.operator.openshift.io
+#tests:
+#  onCreate:
+#    - name: Should be able to create a minimal Etcd
+#      initial: |
+#        apiVersion: operator.openshift.io/v1
+#        kind: Etcd
+#        spec: {} # No spec is required for a Etcd
+#      expected: |
+#        apiVersion: operator.openshift.io/v1
+#        kind: Etcd
+#        spec:
+#          logLevel: Normal
+#          operatorLogLevel: Normal
```

Comment:
You need to comment out only this part. As there is a check that checks this file exists and your integration tests will be failing this now